cpp/unreal_projects/SpearSim/Source/SpearSim/SpearSim.Build.cs

```diff
@@ -9,15 +9,17 @@ public class SpearSim : ModuleRules
     public SpearSim(ReadOnlyTargetRules Target) : base(Target)
     {
         // Disable precompiled headers (in our code but not Unreal code) for faster builds,
-        // easier debugging of compile errors, and strict enforcement of include-what-you-use
+        // easier debugging of compile errors, and strict enforcement of include-what-you-use.
         PCHUsage = ModuleRules.PCHUsageMode.Default;
         PrivatePCHHeaderFile = "";
         bUseUnity = false;
 
-        // Turn off code optimization except in shipping builds for faster build times
+        // Turn off code optimization except in shipping builds for faster build times.
         OptimizeCode = ModuleRules.CodeOptimization.InShippingBuildsOnly;
 
-        // Enable exceptions because some of our third-party dependencies use them
+        // Our ASSERT macro throws exceptions, and so does our templated function Config::get(...),
+        // because it depends yaml-cpp, which throws exceptions. So we need to enable exceptions
```

Comment:
`it depends yaml-cpp` -> `it depends on yaml-cpp`

Reply:
Good catch.